types/react/index.d.ts

```diff
@@ -60,7 +61,7 @@ declare namespace React {
     type Ref<T> = string | { bivarianceHack(instance: T | null): any }["bivarianceHack"] | RefObject<T>;
 
     // tslint:disable-next-line:interface-over-type-literal
```

Comment:
The comment to disable the lint rule is no longer necessary as of this change. Can you remove it?